### PR TITLE
Pure annotations house cleaning: Fix CatmullRomCurve3 formatting

### PR DESCRIPTION
**Pull request by pschroen**
Related issue: #24207

@LeviPesin caught a mistake in my formatting of the last PR. See https://github.com/mrdoob/three.js/pull/24207#discussion_r892076786. 🙏

### CODE REVIEW

**Comment by LeviPesin** [comment]
Can we split this line in three separate like `const px = ...; const py = ...; const pz = ...`?

**Comment by pschroen** [comment]
I had originally done that but then changed it to that format after comparing how other cases were handled, most of the pure annotations are in the multiple variable declarations format.

What changed my mind was the rest of that file is also written as single variable declarations, and also here:
https://github.com/mrdoob/three.js/blob/de17358232c3cc45bb2df0a5a64068cbcdeaa2f8/examples/jsm/utils/CameraUtils.js#L7

Happy to change it to multiple declarations for consistency with the rest of the pure annotations though. 😅

File: src/extras/curves/CatmullRomCurve3.js [hunk 1]
```diff
@@ -79,7 +79,9 @@ function CubicPoly() {
 //
 
 const tmp = /*@__PURE__*/ new Vector3();
-const px = /*@__PURE__*/ new CubicPoly(), /*@__PURE__*/ py = new CubicPoly(), /*@__PURE__*/ pz = new CubicPoly();
+const px = /*@__PURE__*/ new CubicPoly(),
```
Comment (Mugen87) [inline src/extras/curves/CatmullRomCurve3.js:82]: It's okay here but not make a general rule out of this. There are use cases where having everything in a single line improves the readability of the entire file e.g.

https://github.com/mrdoob/three.js/blob/de17358232c3cc45bb2df0a5a64068cbcdeaa2f8/src/renderers/webgl/WebGLUtils.js#L1

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**